server/lib/src/constants/groups.rs

```diff
@@ -407,6 +407,15 @@ lazy_static! {
         ],
         ..Default::default()
     };
+
+    pub static ref BUILTIN_GROUP_APPLICATION_ADMINS: BuiltinGroup = BuiltinGroup {
+        name: "idm_ldap_application_admins",
```

Comment:
just so it's consistent TBH, can we make sure it matches across similar things like `UUID_IDM_APPLICATION_ADMINS` which doesn't match its group name
```suggestion
    pub static ref BUILTIN_GROUP_LDAP_APPLICATION_ADMINS: BuiltinGroup = BuiltinGroup {
        name: "idm_ldap_application_admins",
```